skaffold.yaml

```diff
@@ -60,6 +60,7 @@ deploy:
           enrollments.creds[1].secret: pwd
           enrollments.creds[1].options: "--id.type orderer"
           enrollments.csrHost: network-orderer.orderer
+          toolbox.enabled: true
```

Comment:
if that's the default behavior you might want to cleanup the skaffold file

Reply:
Toolbox is disable by default !
```
toolbox:
  enabled: false
```
The skaffold used here suggests a dev setup, so we enable the toolbox :)

Reply:
Oh right, I got it the other way, thx.